downstream/assemblies/platform/assembly-controller-user-interface.adoc

```diff
@@ -0,0 +1,33 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-controller-user-interface"]
+= {ControllerNameStart} search
+
+:context: controller-UI
+
+= The User Interface
+
+The {ControllerName} _User Interface_ (UI) provides a graphical framework for your IT orchestration requirements. 
+The navigation panel provides quick access to {ControllerName} resources, such as *Projects*, *Inventories*, *Job Templates*, and *Jobs*.
+
+[NOTE]
+====
+The {ControllerName} UI is available for technical preview and is subject to change in future releases. 
+To preview the new UI, click the *Enable Preview of New User Interface* toggle to *On* from the *Miscellaneous System* option of the *Settings* menu.
+
+//image:configure-tower-system-misc-preview-newui.png[image]
+
+After saving, logout and log back in to access the new UI from the preview banner. 
+To return to the current UI, click the link on the top banner where indicated.
+
+//image:ug-dashboard-preview-banner.png[image]
+====
+
+Across the top-right side of the interface, you can access your user profile, the *About* page, view related documentation, and log out. 
```

Comment:
Avoid "top-right", not sure what it should be, maybe Donna can confirm. Something like:
"Access your user profile.... from the menu bar on the header..."